Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -61,8 +62,7 @@
         "IsEnabled": false
       }
     ],
-    "ExcludedExtensions": [],
-    "BearerTokenTimeoutMinutes": ""
```

Comment:
Aren't we using BearerTokenTimeoutMinutes?

Reply:
This one was a duplicate, the actual value is higher up.